### DISCUSSION TITLE
Add docker support to standup mockoon service for automating api's programmatically

### DISCUSSION THREAD

**Discussion by dnmange** [mockoon/mockoon#382] [Ideas] [answered]
This is a great open source project.

I am working on problem where I need to mock the API's and also automate that mocking programmatically, 
Is there a way to add docker support to standup the mockoon service that reads a swagger api's. can anyone guide me if current version already supports that.

**Comment by 255kb** [comment, marked as answer]
hi @dnmange thank you! 

The application is built with Electron and not designed to be run in a headless environment. You can, but it's heavy and a little bit buggy. 
However, rejoice, we are currently working on a [CLI](https://github.com/mockoon/cli) that should be out in a week or two. It will be a NPM package that is able to run a Mockoon export file. Concerning the Docker file, it will be added soon after and there is already an [issue](https://github.com/mockoon/cli/issues/2) opened.
